### ISSUE TITLE
Error running sync step MySQL (0.30.4)

### ISSUE THREAD

**Issue by Crusader4Christ** [metabase/metabase#9977]
```
 ... 5 more
 at org.quartz.impl.jdbcjobstore.JobStoreSupport.storeTrigger(JobStoreSupport.java:1194)
 at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.selectJobDetail(StdJDBCDelegate.java:873)
 at metabase.task.DynamicClassLoadHelper.loadClass(Unknown Source)
 at metabase.task.DynamicClassLoadHelper$_loadClass.invoke(DynamicClassLoadHelper.clj:38)
 at metabase.task.DynamicClassLoadHelper$_loadClass.invokeStatic(DynamicClassLoadHelper.clj:46)
 at metabase.task.DynamicClassLoadHelper$require_task_namespace.invoke(DynamicClassLoadHelper.clj:29)
 at metabase.task.DynamicClassLoadHelper$require_task_namespace.invokeStatic(DynamicClassLoadHelper.clj:36)
 at clojure.lang.RestFn.invoke(RestFn.java:408)
 at clojure.core$require.doInvoke(core.clj:5947)
 at clojure.core$require.invokeStatic(core.clj:5947)
 at clojure.core$apply.invokeStatic(core.clj:659)
 at clojure.lang.RestFn.applyTo(RestFn.java:137)
 at clojure.core$load_libs.doInvoke(core.clj:5909)
 at clojure.core$load_libs.invokeStatic(core.clj:5925)
 at clojure.core$load_lib.invokeStatic(core.clj:5887)
 at clojure.lang.RestFn.applyTo(RestFn.java:142)
 at clojure.core$load_lib.doInvoke(core.clj:5868)
 at clojure.core$apply.invokeStatic(core.clj:659)
 at clojure.core$load_one.invokeStatic(core.clj:5848)
 at clojure.lang.RestFn.invoke(RestFn.java:408)
 at clojure.core$load_lib$fn__6493.invoke(core.clj:5888)
 at clojure.core$load_one.invoke(core.clj:5843)
 at clojure.core$load.doInvoke(core.clj:6029)
 at clojure.core$load$fn__6548.invoke(core.clj:6046)
 at clojure.core$load.invokeStatic(core.clj:6045)
 at clojure.lang.RT.load(RT.java:426)
 Caused by: java.io.FileNotFoundException: Could not locate metabase/task/task_history_cleanup__init.class or metabase/task/task_history_cleanup.clj on classpath. Please check that namespaces with dashes use underscores in the Clojure file name.
 
 at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.manage(JobStoreSupport.java:3934)
 at clojure.lang.RT.load(RT.java:463)
 at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.run(JobStoreSupport.java:3955)
 at org.quartz.impl.jdbcjobstore.JobStoreSupport.doUpdateOfMisfiredTrigger(JobStoreSupport.java:1026)
 at org.quartz.impl.jdbcjobstore.JobStoreSupport.recoverMisfiredJobs(JobStoreSupport.java:975)
 at org.quartz.impl.jdbcjobstore.JobStoreSupport.doRecoverMisfires(JobStoreSupport.java:3174)
 at org.quartz.impl.jdbcjobstore.JobStoreSupport.storeTrigger(JobStoreSupport.java:1212)
 at org.quartz.JobPersistenceException: Couldn't store trigger 'DEFAULT.metabase.task.task-history-cleanup.trigger' for 'DEFAULT.metabase.task.task-history-cleanup.job' job:Could not locate metabase/task/task_history_cleanup__init.class or metabase/task/task_history_cleanup.clj on classpath. Please check that namespaces with dashes use underscores in the Clojure file name. [See nested exception: java.io.FileNotFoundException: Could not locate metabase/task/task_history_cleanup__init.class or metabase/task/task_history_cleanup.clj on classpath. Please check that namespaces with dashes use underscores in the Clojure file name.]
	2019-15-05, 04:57:11.039 O2U1uWoBEXIGVWvjLTZw	05-15 04:57:11 [1mERROR jdbcjobstore.JobStoreTX[0m :: MisfireHandler: Error handling misfires: Couldn't store trigger 'DEFAULT.metabase.task.task-history-cleanup.trigger' for 'DEFAULT.metabase.task.task-history-cleanup.job' job:Could not locate metabase/task/task_history_cleanup__init.class or metabase/task/task_history_cleanup.clj on classpath. Please check that namespaces with dashes use underscores in the Clojure file name.
```

- Your operating system: Ubuntu
- Your databases: MySQL
- Metabase version: v0.30.4
- Metabase hosting environment: Docker
- Metabase internal database: MySQL

**Comment by camsaul** [comment]
Hi, don't forget that you can use the search bar to see if an issue was previously reported. We already had an issue for this: #8941

This was fixed in #9671 which shipped as part of 0.32.2

**Comment by camsaul** [comment]
Duplicate of #8941

**Comment by Crusader4Christ** [comment]
@camsaul Can this PR be merged to 0.30.4 because it is last stable working version?
